Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -403,6 +432,7 @@ public void run() {
         }
       }
     }
+    myFilePathsWithUnsentChanges.clear();;
```

Comment:
Actually you're right, it is better to put it into UpdateContentConsumer.onResponse() inside updateFilesContent().